### PR TITLE
initialize vault in workflow

### PR DESCRIPTION
**Pull request by trriplejay**
#84

results in:
image

### CODE REVIEW

**Comment by trriplejay** [comment]
Hmm. Left some extra stuff in the pr. Will fix in a bit.

**Comment by ric03uec** [comment]
@trriplejay rebase required

**Comment by ric03uec** [comment]
i think we should remove the cleanup file and add it as a command in installer. something like 
`./admiral.sh clean`
this should ask for prompt and remove containers, folders and any configs

File: api/workflow/initialize.js [hunk 1]
```diff
@@ -53,3 +55,56 @@ function _initializeDatabase(bag, next) {
     }
   );
 }
+
+function _getSystemConfigs(bag, next) {
```
Comment (trriplejay) [inline api/workflow/initialize.js]: Will remove this.

**Pull request merged**